core/src/com/unciv/ui/screens/devconsole/ConsoleTileCommands.kt

```diff
@@ -84,10 +93,47 @@ class ConsoleTileCommands: ConsoleCommandNode {
             DevConsoleResponse.OK
         },
 
+        "addriver" to ConsoleRiverAction("tile addriver <direction>", true),
+        "removeriver" to ConsoleRiverAction("tile removeriver <direction>", false),
     )
 
     private fun getTerrainFeature(console: DevConsolePopup, param: String) =
         console.gameInfo.ruleset.terrains.values.asSequence()
         .filter { it.type == TerrainType.TerrainFeature }.findCliInput(param)
         ?: throw ConsoleErrorException("Unknown feature")
+
+    private enum class RiverDirections(val clockPosition: Int) {
```

Comment:
This is a good idea, we should move this to where the other river logic is because it will make things more readable